awscdk-minecraft/src/zip-directory.py

```diff
@@ -0,0 +1,40 @@
+import os
+from zipfile import ZipFile
+
+def collect_file_paths(target_dir):
+    # Initialize a list for adding filepath strings to
+    file_paths = []
```

Comment:
For this entire function, can I suggest the following?

```python
target_dir = Path(target_dir)
file_paths: List[Path] = list(target_dir.rglob("*"))
return file_paths
```

Reply:
You'd change the return type in @mathematicalmichael to `List[Path]` as well for this, which will make `pathlib` fans like me happy.

Reply:
That's pretty cool @phitoduck